diff -Nur krb5-1.3.1.ori/src/configure.in krb5-1.3.1.mod/src/configure.in
--- krb5-1.3.1.ori/src/configure.in	2003-01-11 00:18:53.000000000 +0100
+++ krb5-1.3.1.mod/src/configure.in	2006-11-04 01:25:20.000000000 +0100
@@ -3,10 +3,12 @@
 #
 # autoconf 2.49 defaults to a /dev/null cache file, which is what we
 # do not want for performance reasons. 
-if test "x$cache_file" = "x/dev/null"; then
-  cache_file=./config.cache
-  AC_CACHE_LOAD
-fi
+# but using that trick makes many problems when there are spaces inside
+# FLAGS (ie rpmflags)
+# if test "x$cache_file" = "x/dev/null"; then
+#  cache_file=./config.cache
+#  AC_CACHE_LOAD
+# fi
 
 CONFIG_RULES
 
