Michael Calmer's fix for a crash bug, RT #5394.

--- src/lib/krb5/os/sendto_kdc.c
+++ src/lib/krb5/os/sendto_kdc.c	2007/01/17 14:17:10
@@ -1100,7 +1100,7 @@
                 struct sockaddr *remoteaddr, socklen_t *remoteaddrlen,
 		int *addr_used)
 {
-    int i, pass;
+    int i = 0, pass;
     int delay_this_pass = 2;
     krb5_error_code retval;
     struct conn_state *conns;
@@ -1135,7 +1135,7 @@
 	    return ENOMEM;
 	}
 
-	memset(conns, 0, n_conns * sizeof(callback_data[i]));
+	memset(callback_data, 0, n_conns * sizeof(callback_data[i]));
     }
 
     for (i = 0; i < n_conns; i++) {
