From Kevin Coffman, via the nfs4 mailing list.
properly export the acceptor subkey when creating a lucid context
diff -up src/lib/gssapi/krb5/lucid_context.c ./src/lib/gssapi/krb5/lucid_context.c
--- src/lib/gssapi/krb5/lucid_context.c	2008-04-01 16:28:11.000000000 -0400
+++ src/lib/gssapi/krb5/lucid_context.c	2008-04-01 16:28:01.000000000 -0400
@@ -231,7 +231,7 @@ make_external_lucid_ctx_v1(
 	    				&lctx->cfx_kd.ctx_key)))
 	    goto error_out;
 	if (gctx->have_acceptor_subkey) {
-	    if ((retval = copy_keyblock_to_lucid_key(gctx->enc,
+	    if ((retval = copy_keyblock_to_lucid_key(gctx->acceptor_subkey,
 	    				&lctx->cfx_kd.acceptor_subkey)))
 		goto error_out;
 	    lctx->cfx_kd.have_acceptor_subkey = 1;
