--- krb5-1.15/src/tests/resolve/Makefile.in.orig	2017-02-16 22:16:20.209242926 +0100
+++ krb5-1.15/src/tests/resolve/Makefile.in	2017-02-16 22:20:24.069240143 +0100
@@ -17,9 +17,9 @@
 	$(CC_LINK) -o $@ fake-addrinfo-test.o $(SUPPORT_LIB) $(LIBS)
 
 check: resolve addrinfo-test fake-addrinfo-test
-	$(RUN_TEST) ./resolve
-	$(RUN_TEST) ./addrinfo-test -p telnet
-	$(RUN_TEST) ./fake-addrinfo-test -p telnet
+	if [ "$(OFFLINE)" = no ]; then $(RUN_TEST) ./resolve ; fi
+	if [ "$(OFFLINE)" = no ]; then $(RUN_TEST) ./addrinfo-test -p telnet ; fi
+	if [ "$(OFFLINE)" = no ]; then $(RUN_TEST) ./fake-addrinfo-test -p telnet ; fi
 
 install:
 
--- krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test~	2007-11-21 17:52:04.000000000 +0100
+++ krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test	2007-11-21 17:58:16.000000000 +0100
@@ -12,17 +12,7 @@
 	TMP2=${TMPDIR-.}/t2
 	TMP3=${TMPDIR-.}/t3
 
-	if [ \! -z "$WORDLIST" -a -f "$WORDLIST" ]; then
-		DICT=$WORDLIST
-	elif [ -f /usr/local/lib/dict/words ]; then
-		DICT=/usr/local/lib/dict/words
-	elif [ -f /usr/share/dict/words ]; then
-		DICT=/usr/share/dict/words
-	elif [ -f /usr/dict/words ]; then
-		DICT=/usr/dict/words
-	elif [ -f /usr/share/lib/dict/words ]; then
-		DICT=/usr/share/lib/dict/words
-	elif [ -f $srcdir/../test/dictionary ]; then
+	if [ -f $srcdir/../test/dictionary ]; then
 		DICT=`cd $srcdir/../test && pwd`/dictionary
 	else
 		echo 'run.test: no dictionary'
--- krb5-1.15/src/lib/krb5/krb/Makefile.in.orig	2017-02-16 22:21:37.502572637 +0100
+++ krb5-1.15/src/lib/krb5/krb/Makefile.in	2017-02-16 22:22:42.059238568 +0100
@@ -498,9 +498,11 @@
 	$(RUN_TEST) ./t_sname_match
 
 check-pytests: t_expire_warn t_vfy_increds
-	$(RUNPYTEST) $(srcdir)/t_expire_warn.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_vfy_increds.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_in_ccache_patypes.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_expire_warn.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_vfy_increds.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_in_ccache_patypes.py $(PYTESTFLAGS) ; \
+	fi
 
 check-cmocka: t_parse_host_string
 	$(RUN_TEST) ./t_parse_host_string > /dev/null
--- krb5-1.15/src/kdc/Makefile.in.orig	2017-02-16 22:23:21.009238123 +0100
+++ krb5-1.15/src/kdc/Makefile.in	2017-02-18 08:23:41.544506963 +0100
@@ -83,8 +83,10 @@
 	$(RUN_TEST) ./t_replay > /dev/null
 
 check-pytests:
-	$(RUNPYTEST) $(srcdir)/t_workers.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_emptytgt.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_workers.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_emptytgt.py $(PYTESTFLAGS) ; \
+	fi
 
 install:
 	$(INSTALL_PROGRAM) krb5kdc ${DESTDIR}$(SERVER_BINDIR)/krb5kdc
--- krb5-1.15/src/appl/gss-sample/Makefile.in.orig	2017-02-18 08:24:33.754506368 +0100
+++ krb5-1.15/src/appl/gss-sample/Makefile.in	2017-02-18 08:35:02.454499191 +0100
@@ -43,7 +43,9 @@
 	$(RM) gss-server gss-client
 
 check-pytests:
-	$(RUNPYTEST) $(srcdir)/t_gss_sample.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_gss_sample.py $(PYTESTFLAGS) ; \
+	fi
 
 install-unix:
 	$(INSTALL_PROGRAM) gss-client $(DESTDIR)$(CLIENT_BINDIR)/gss-client
--- krb5-1.15/src/tests/gssapi/Makefile.in.orig	2017-02-18 08:35:55.481165250 +0100
+++ krb5-1.15/src/tests/gssapi/Makefile.in	2017-02-18 08:37:07.877831093 +0100
@@ -44,13 +44,15 @@
 	t_enctypes t_err t_export_cred t_export_name t_imp_cred t_inq_cred \
 	t_inq_ctx t_inq_mechs_name t_iov t_pcontok t_s4u t_s4u2proxy_krb5 \
 	t_spnego t_srcattrs
-	$(RUNPYTEST) $(srcdir)/t_gssapi.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_ccselect.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_client_keytab.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_enctypes.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_export_cred.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_s4u.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_authind.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_gssapi.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_ccselect.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_client_keytab.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_enctypes.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_export_cred.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_s4u.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_authind.py $(PYTESTFLAGS) ; \
+	fi
 
 ccinit: ccinit.o $(KRB5_BASE_DEPLIBS)
 	$(CC_LINK) -o ccinit ccinit.o $(KRB5_BASE_LIBS)
#--- krb5-1.9.1/src/tests/mkeystash_compat/Makefile.in~	2010-12-03 01:05:44.000000000 +0100
#+++ krb5-1.9.1/src/tests/mkeystash_compat/Makefile.in	2011-08-03 12:55:23.507781811 +0200
#@@ -37,14 +37,16 @@
# 
# # Verify that the mkey stash code is backward compat with old/non-keytab stashfile format
# mkeystash_check: kdc.conf krb5.conf bigendian
#-	$(RM) $(TEST_DB)* stash_file
#-	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) create -s -W
#-	# overwrite keytab stash file with old format stash, depends on endianness of current test system
#-	./bigendian && cp $(srcdir)/old_stash_bendian stash_file || cp $(srcdir)/old_stash_lendian stash_file
#-	# getprinc will fail if old stash file can not be read
#-	$(RUN_SETUP) $(VALGRIND) ../../kadmin/cli/kadmin.local $(KADMIN_OPTS) -q 'getprinc K/M'
#-	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) destroy -f
#-	$(RM) $(TEST_DB)* stash_file
#+	if [ "$(OFFLINE)" = no ]; then \
#+	$(RM) $(TEST_DB)* stash_file && \
#+	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) create -s -W && \
#+	# overwrite keytab stash file with old format stash, depends on endianness of current test system && \
#+	./bigendian && cp $(srcdir)/old_stash_bendian stash_file || cp $(srcdir)/old_stash_lendian stash_file && \
#+	# getprinc will fail if old stash file can not be read && \
#+	$(RUN_SETUP) $(VALGRIND) ../../kadmin/cli/kadmin.local $(KADMIN_OPTS) -q 'getprinc K/M' && \
#+	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) destroy -f && \
#+	$(RM) $(TEST_DB)* stash_file ; \
#+	fi
# 
# clean::
# 	$(RM) kdc.conf krb5.conf bigendian.$(OBJEXT) bigendian
--- krb5-1.15/src/tests/Makefile.in.orig	2017-02-18 08:43:08.467826973 +0100
+++ krb5-1.15/src/tests/Makefile.in	2017-02-18 09:09:28.181142270 +0100
@@ -89,84 +89,88 @@
 	mv krb5.new krb5.conf
 
 kdb_check: kdc.conf krb5.conf
-	$(RM) $(TEST_DB)*
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W
-	$(RUN_DB_TEST) ../tests/create/kdb5_mkdums $(KTEST_OPTS)
-	$(RUN_DB_TEST) ../tests/verify/kdb5_verify $(KTEST_OPTS)
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f
-	@echo "====> NOTE!"
-	@echo "The following 'create' command is needed due to a change"
-	@echo "in functionality caused by DAL integration.  See ticket 3973."
-	@echo ====
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load $(TEST_DB).dump
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load -update -ov $(TEST_DB).ovdump
-	$(RUN_DB_TEST) ../tests/verify/kdb5_verify $(KTEST_OPTS)
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump2
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump2
-	sort $(TEST_DB).dump > $(TEST_DB).sort
-	sort $(TEST_DB).dump2 > $(TEST_DB).sort2
-	sort $(TEST_DB).ovdump > $(TEST_DB).ovsort
-	sort $(TEST_DB).ovdump2 > $(TEST_DB).ovsort2
-	cmp $(TEST_DB).sort $(TEST_DB).sort2
-	cmp $(TEST_DB).ovsort $(TEST_DB).ovsort2
-	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f
-	$(RM) $(TEST_DB)* stash_file
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RM) $(TEST_DB)* && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W && \
+	$(RUN_DB_TEST) ../tests/create/kdb5_mkdums $(KTEST_OPTS) && \
+	$(RUN_DB_TEST) ../tests/verify/kdb5_verify $(KTEST_OPTS) && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f && \
+	@echo "====> NOTE!" ; \
+	@echo "The following 'create' command is needed due to a change" ; \
+	@echo "in functionality caused by DAL integration.  See ticket 3973." ; \
+	@echo ==== ; \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load $(TEST_DB).dump && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load -update -ov $(TEST_DB).ovdump && \
+	$(RUN_DB_TEST) ../tests/verify/kdb5_verify $(KTEST_OPTS) && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump2 && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump2 && \
+	sort $(TEST_DB).dump > $(TEST_DB).sort && \
+	sort $(TEST_DB).dump2 > $(TEST_DB).sort2 && \
+	sort $(TEST_DB).ovdump > $(TEST_DB).ovsort && \
+	sort $(TEST_DB).ovdump2 > $(TEST_DB).ovsort2 && \
+	cmp $(TEST_DB).sort $(TEST_DB).sort2 && \
+	cmp $(TEST_DB).ovsort $(TEST_DB).ovsort2 && \
+	$(RUN_DB_TEST) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f && \
+	$(RM) $(TEST_DB)* stash_file ; \
+	fi
 
 check-pytests: adata etinfo forward gcred hist hooks hrealm icred kdbtest
 check-pytests: localauth plugorder rdreq responder s2p s4u2proxy unlockiter
-	$(RUNPYTEST) $(srcdir)/t_general.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_hooks.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_dump.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_iprop.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kprop.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_policy.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_changepw.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_pkinit.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_otp.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_localauth.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kadm5_hook.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_pwqual.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_hostrealm.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kdb_locking.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_keyrollover.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_renew.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_renprinc.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_ccache.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_stringattr.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_sesskeynego.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_crossrealm.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_referral.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_skew.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_keytab.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kadmin_acl.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kadmin_parsing.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kdb.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_keydata.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_mkey.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_rdreq.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_sn2princ.py $(PYTESTFLAGS) $(OFFLINE)
-	$(RUNPYTEST) $(srcdir)/t_cve-2012-1014.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_cve-2012-1015.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_cve-2013-1416.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_cve-2013-1417.py $(PYTESTFLAGS)
-	$(RM) au.log
-	$(RUNPYTEST) $(srcdir)/t_audit.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_general.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_hooks.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_dump.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_iprop.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kprop.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_policy.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_changepw.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_pkinit.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_otp.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_localauth.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kadm5_hook.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_pwqual.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_hostrealm.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kdb_locking.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_keyrollover.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_renew.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_renprinc.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_ccache.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_stringattr.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_sesskeynego.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_crossrealm.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_referral.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_skew.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_keytab.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kadmin_acl.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kadmin_parsing.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kdb.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_keydata.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_mkey.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_rdreq.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_sn2princ.py $(PYTESTFLAGS) $(OFFLINE) && \
+	$(RUNPYTEST) $(srcdir)/t_cve-2012-1014.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_cve-2012-1015.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_cve-2013-1416.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_cve-2013-1417.py $(PYTESTFLAGS) && \
+	$(RM) au.log && \
+	$(RUNPYTEST) $(srcdir)/t_audit.py $(PYTESTFLAGS) && \
 	$(RUNPYTEST) $(srcdir)/jsonwalker.py -d $(srcdir)/au_dict.json \
-			-i au.log
-	$(RUNPYTEST) $(srcdir)/t_salt.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_etype_info.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_bogus_kdc_req.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kdc_log.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_proxy.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_unlockiter.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_errmsg.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_authdata.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_preauth.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_princflags.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_tabdump.py $(PYTESTFLAGS)
+			-i au.log && \
+	$(RUNPYTEST) $(srcdir)/t_salt.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_etype_info.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_bogus_kdc_req.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kdc_log.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_proxy.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_unlockiter.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_errmsg.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_authdata.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_preauth.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_princflags.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_tabdump.py $(PYTESTFLAGS) ; \
+	fi
 
 clean:
 	$(RM) adata etinfo forward gcred hist hooks hrealm icred kdbtest
