--- krb5-1.6/src/tests/resolve/Makefile.in~	2006-10-14 01:54:24.000000000 +0200
+++ krb5-1.6/src/tests/resolve/Makefile.in	2007-03-31 13:19:53.138858011 +0200
@@ -22,9 +22,9 @@
 	$(CC_LINK) -o $@ fake-addrinfo-test.o $(SUPPORT_LIB) $(LIBS)
 
 check:: resolve addrinfo-test fake-addrinfo-test
-	$(RUN_SETUP) $(VALGRIND) ./resolve
-	$(RUN_SETUP) $(VALGRIND) ./addrinfo-test -p telnet
-	$(RUN_SETUP) $(VALGRIND) ./fake-addrinfo-test -p telnet
+	if [ "$(OFFLINE)" = no ]; then $(RUN_SETUP) $(VALGRIND) ./resolve ep09.pld-linux.org ; fi
+	if [ "$(OFFLINE)" = no ]; then $(RUN_SETUP) $(VALGRIND) ./addrinfo-test -p telnet ; fi
+	if [ "$(OFFLINE)" = no ]; then $(RUN_SETUP) $(VALGRIND) ./fake-addrinfo-test -p telnet ; fi
 
 install::
 
--- krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test~	2007-11-21 17:52:04.000000000 +0100
+++ krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test	2007-11-21 17:58:16.000000000 +0100
@@ -12,17 +12,7 @@
 	TMP2=${TMPDIR-.}/t2
 	TMP3=${TMPDIR-.}/t3
 
-	if [ \! -z "$WORDLIST" -a -f "$WORDLIST" ]; then
-		DICT=$WORDLIST
-	elif [ -f /usr/local/lib/dict/words ]; then
-		DICT=/usr/local/lib/dict/words
-	elif [ -f /usr/share/dict/words ]; then
-		DICT=/usr/share/dict/words
-	elif [ -f /usr/dict/words ]; then
-		DICT=/usr/dict/words
-	elif [ -f /usr/share/lib/dict/words ]; then
-		DICT=/usr/share/lib/dict/words
-	elif [ -f $srcdir/../test/dictionary ]; then
+	if [ -f $srcdir/../test/dictionary ]; then
 		DICT=`cd $srcdir/../test && pwd`/dictionary
 	else
 		echo 'run.test: no dictionary'
--- krb5-1.9.1/src/lib/krb5/krb/Makefile.in~	2010-12-03 01:05:44.000000000 +0100
+++ krb5-1.9.1/src/lib/krb5/krb/Makefile.in	2011-08-03 00:38:37.538243651 +0200
@@ -431,7 +431,9 @@
 	$(RUN_SETUP) $(VALGRIND) ./t_etypes
 
 check-pytests:: t_expire_warn
-	$(RUNPYTEST) $(srcdir)/t_expire_warn.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_expire_warn.py $(PYTESTFLAGS) ; \
+	fi
 
 clean::
 	$(RM) $(OUTPRE)t_walk_rtree$(EXEEXT) $(OUTPRE)t_walk_rtree.$(OBJEXT) \
--- krb5-1.9.1/src/kdc/Makefile.in~	2010-11-01 20:49:29.000000000 +0100
+++ krb5-1.9.1/src/kdc/Makefile.in	2011-08-03 00:50:14.395525895 +0200
@@ -66,7 +66,9 @@
 	$(RM) test.out
 
 check-pytests::
-	$(RUNPYTEST) $(srcdir)/t_workers.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_workers.py $(PYTESTFLAGS) ; \
+	fi
 
 install::
 	$(INSTALL_PROGRAM) krb5kdc ${DESTDIR}$(SERVER_BINDIR)/krb5kdc
--- krb5-1.9.1/src/appl/gss-sample/Makefile.in~	2010-12-03 01:05:44.000000000 +0100
+++ krb5-1.9.1/src/appl/gss-sample/Makefile.in	2011-08-03 01:03:37.904570242 +0200
@@ -45,7 +45,9 @@
 	$(RM) gss-server gss-client
 
 check-pytests::
-	$(RUNPYTEST) $(srcdir)/t_gss_sample.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_gss_sample.py $(PYTESTFLAGS) ; \
+	fi
 
 install-unix::
 	$(INSTALL_PROGRAM) gss-client $(DESTDIR)$(CLIENT_BINDIR)/gss-client
--- krb5-1.9.1/src/tests/Makefile.in~	2010-12-15 00:10:42.000000000 +0100
+++ krb5-1.9.1/src/tests/Makefile.in	2011-08-03 01:13:45.330962610 +0200
@@ -61,12 +61,14 @@
 	$(RM) $(TEST_DB)* stash_file
 
 check-pytests::
-	$(RUNPYTEST) $(srcdir)/t_general.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_anonpkinit.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_lockout.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_kadm5_hook.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_keyrollover.py $(PYTESTFLAGS)
-	$(RUNPYTEST) $(srcdir)/t_renew.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_general.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_anonpkinit.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_lockout.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_kadm5_hook.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_keyrollover.py $(PYTESTFLAGS) && \
+	$(RUNPYTEST) $(srcdir)/t_renew.py $(PYTESTFLAGS) ; \
+	fi
 
 clean::
 	$(RM) kdc.conf
--- krb5-1.9.1/src/tests/gssapi/Makefile.in~	2010-12-03 01:05:44.000000000 +0100
+++ krb5-1.9.1/src/tests/gssapi/Makefile.in	2011-08-03 01:15:11.187294436 +0200
@@ -11,7 +11,9 @@
 all:: t_imp_name t_s4u t_namingexts t_gssexts t_spnego t_saslname
 
 check-pytests:: t_spnego
-	$(RUNPYTEST) $(srcdir)/t_gssapi.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_gssapi.py $(PYTESTFLAGS) ; \
+	fi
 
 t_imp_name: t_imp_name.o $(GSS_DEPLIBS) $(KRB5_BASE_DEPLIBS)
 	$(CC_LINK) -o t_imp_name t_imp_name.o $(GSS_LIBS) $(KRB5_BASE_LIBS)
--- krb5-1.9.1/src/tests/mkeystash_compat/Makefile.in~	2010-12-03 01:05:44.000000000 +0100
+++ krb5-1.9.1/src/tests/mkeystash_compat/Makefile.in	2011-08-03 12:55:23.507781811 +0200
@@ -37,14 +37,16 @@
 
 # Verify that the mkey stash code is backward compat with old/non-keytab stashfile format
 mkeystash_check: kdc.conf krb5.conf bigendian
-	$(RM) $(TEST_DB)* stash_file
-	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) create -s -W
-	# overwrite keytab stash file with old format stash, depends on endianness of current test system
-	./bigendian && cp $(srcdir)/old_stash_bendian stash_file || cp $(srcdir)/old_stash_lendian stash_file
-	# getprinc will fail if old stash file can not be read
-	$(RUN_SETUP) $(VALGRIND) ../../kadmin/cli/kadmin.local $(KADMIN_OPTS) -q 'getprinc K/M'
-	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) destroy -f
-	$(RM) $(TEST_DB)* stash_file
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RM) $(TEST_DB)* stash_file && \
+	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) create -s -W && \
+	# overwrite keytab stash file with old format stash, depends on endianness of current test system && \
+	./bigendian && cp $(srcdir)/old_stash_bendian stash_file || cp $(srcdir)/old_stash_lendian stash_file && \
+	# getprinc will fail if old stash file can not be read && \
+	$(RUN_SETUP) $(VALGRIND) ../../kadmin/cli/kadmin.local $(KADMIN_OPTS) -q 'getprinc K/M' && \
+	$(RUN_SETUP) $(VALGRIND) ../../kadmin/dbutil/kdb5_util $(KDB_OPTS) destroy -f && \
+	$(RM) $(TEST_DB)* stash_file ; \
+	fi
 
 clean::
 	$(RM) kdc.conf krb5.conf bigendian.$(OBJEXT) bigendian
--- krb5-1.9.1/src/tests/Makefile.in~	2011-08-03 12:52:47.476781846 +0200
+++ krb5-1.9.1/src/tests/Makefile.in	2011-08-03 12:56:18.963781798 +0200
@@ -34,31 +34,33 @@
 	mv krb5.new krb5.conf
 
 kdb_check: kdc.conf krb5.conf
-	$(RM) $(TEST_DB)*
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W
-	$(RUN_SETUP) $(VALGRIND) ../tests/create/kdb5_mkdums $(KTEST_OPTS) 
-	$(RUN_SETUP) $(VALGRIND) ../tests/verify/kdb5_verify $(KTEST_OPTS) 
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f
-	@echo "====> NOTE!"
-	@echo "The following 'create' command is needed due to a change"
-	@echo "in functionality caused by DAL integration.  See ticket 3973."
-	@echo ====
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load $(TEST_DB).dump 
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load -update -ov $(TEST_DB).ovdump 
-	$(RUN_SETUP) $(VALGRIND) ../tests/verify/kdb5_verify $(KTEST_OPTS) 
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump2
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump2
-	sort $(TEST_DB).dump > $(TEST_DB).sort
-	sort $(TEST_DB).dump2 > $(TEST_DB).sort2
-	sort $(TEST_DB).ovdump > $(TEST_DB).ovsort
-	sort $(TEST_DB).ovdump2 > $(TEST_DB).ovsort2
-	cmp $(TEST_DB).sort $(TEST_DB).sort2
-	cmp $(TEST_DB).ovsort $(TEST_DB).ovsort2
-	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f
-	$(RM) $(TEST_DB)* stash_file
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RM) $(TEST_DB)* && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W && \
+	$(RUN_SETUP) $(VALGRIND) ../tests/create/kdb5_mkdums $(KTEST_OPTS)  && \
+	$(RUN_SETUP) $(VALGRIND) ../tests/verify/kdb5_verify $(KTEST_OPTS)  && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f && \
+	@echo "====> NOTE!" ;\
+	@echo "The following 'create' command is needed due to a change" ;\
+	@echo "in functionality caused by DAL integration.  See ticket 3973." ;\
+	@echo ==== ;\
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) create -W && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load $(TEST_DB).dump  && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) load -update -ov $(TEST_DB).ovdump  && \
+	$(RUN_SETUP) $(VALGRIND) ../tests/verify/kdb5_verify $(KTEST_OPTS)  && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump $(TEST_DB).dump2 && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) dump -ov $(TEST_DB).ovdump2 && \
+	sort $(TEST_DB).dump > $(TEST_DB).sort && \
+	sort $(TEST_DB).dump2 > $(TEST_DB).sort2 && \
+	sort $(TEST_DB).ovdump > $(TEST_DB).ovsort && \
+	sort $(TEST_DB).ovdump2 > $(TEST_DB).ovsort2 && \
+	cmp $(TEST_DB).sort $(TEST_DB).sort2 && \
+	cmp $(TEST_DB).ovsort $(TEST_DB).ovsort2 && \
+	$(RUN_SETUP) $(VALGRIND) ../kadmin/dbutil/kdb5_util $(KADMIN_OPTS) destroy -f && \
+	$(RM) $(TEST_DB)* stash_file ;\
+	fi
 
 check-pytests::
 	if [ "$(OFFLINE)" = no ]; then \
