--- krb5-1.6/src/tests/resolve/Makefile.in~	2006-10-14 01:54:24.000000000 +0200
+++ krb5-1.6/src/tests/resolve/Makefile.in	2007-03-31 13:19:53.138858011 +0200
@@ -22,7 +22,7 @@
 	$(CC_LINK) -o $@ fake-addrinfo-test.o $(SUPPORT_LIB) $(LIBS)
 
 check:: resolve addrinfo-test
-	$(RUN_SETUP) $(VALGRIND) ./resolve
+	$(RUN_SETUP) $(VALGRIND) ./resolve www.pld-linux.org
 	$(RUN_SETUP) $(VALGRIND) ./addrinfo-test -p telnet
 	$(RUN_SETUP) $(VALGRIND) ./fake-addrinfo-test -p telnet
 
--- krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test~	2007-11-21 17:52:04.000000000 +0100
+++ krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test	2007-11-21 17:58:16.000000000 +0100
@@ -12,17 +12,7 @@
 	TMP2=${TMPDIR-.}/t2
 	TMP3=${TMPDIR-.}/t3
 
-	if [ \! -z "$WORDLIST" -a -f "$WORDLIST" ]; then
-		DICT=$WORDLIST
-	elif [ -f /usr/local/lib/dict/words ]; then
-		DICT=/usr/local/lib/dict/words
-	elif [ -f /usr/share/dict/words ]; then
-		DICT=/usr/share/dict/words
-	elif [ -f /usr/dict/words ]; then
-		DICT=/usr/dict/words
-	elif [ -f /usr/share/lib/dict/words ]; then
-		DICT=/usr/share/lib/dict/words
-	elif [ -f $srcdir/../test/dictionary ]; then
+	if [ -f $srcdir/../test/dictionary ]; then
 		DICT=`cd $srcdir/../test && pwd`/dictionary
 	else
 		echo 'run.test: no dictionary'
@@ -404,9 +404,9 @@
 	awk 'BEGIN {
 		for (i = 1; i <= 10; ++i) {
 			printf("p\nkkey1\nD/bin/sh\n");
-			printf("p\nkkey2\nD/bin/csh\n");
+			printf("p\nkkey2\nD/bin/awk\n");
 			if (i % 8 == 0) {
-				printf("c\nkkey2\nD/bin/csh\n");
+				printf("c\nkkey2\nD/bin/awk\n");
 				printf("c\nkkey1\nD/bin/sh\n");
 				printf("e\t%d of 10 (comparison)\n", i);
 			} else
