diff -Nur krb5-1.3.1.ori/src/configure.in krb5-1.3.1.mod/src/configure.in
--- krb5-1.3.1.ori/src/configure.in	2003-01-11 00:18:53.000000000 +0100
+++ krb5-1.3.1.mod/src/configure.in	2006-11-04 01:25:20.000000000 +0100
@@ -3,10 +3,12 @@
 dnl
 dnl autoconf 2.49 defaults to a /dev/null cache file, which is what we
 dnl do not want for performance reasons. 
-if test "x$cache_file" = "x/dev/null"; then
-  cache_file=./config.cache
-  AC_CACHE_LOAD
-fi
+dnl but using that trick makes many problems when there are spaces inside
+dnl FLAGS (ie rpmflags)
+dnl if test "x$cache_file" = "x/dev/null"; then
+dnl  cache_file=./config.cache
+dnl  AC_CACHE_LOAD
+dnl fi
 dnl
 CONFIG_RULES
 dnl
@@ -75,5 +77,6 @@
   AC_CONFIG_SUBDIRS(krb524)
 fi
 AC_CONFIG_SUBDIRS(kdc kadmin slave clients appl tests)
-AC_CONFIG_FILES(krb5-config, [chmod +x krb5-config])
+dnl AC_CONFIG_FILES(krb5-config, [chmod +x krb5-config])
+AC_CONFIG_FILES(krb5-config)
 V5_AC_OUTPUT_MAKEFILE(. util util/send-pr lib config-files gen-manpages)
