--- krb5-1.6/src/appl/telnet/configure.in~	2006-03-28 06:35:02.000000000 +0200
+++ krb5-1.6/src/appl/telnet/configure.in	2007-03-30 19:36:45.321386958 +0200
@@ -52,12 +52,16 @@
 dnl
 AC_CHECK_LIB(termcap,main,AC_DEFINE(TERMCAP,1,[Define if termcap library is available])
 LIBS="$LIBS -ltermcap")
-AC_CHECK_LIB(curses,setupterm,LIBS="$LIBS -lcurses",
-  AC_CHECK_LIB(ncurses,setupterm,LIBS="$LIBS -lncurses")
-)
+old_CFLAGS="$CFLAGS"
+old_LDFLAGS="$LDFLAGS"
+CFLAGS="-I/usr/include/ncurses"
+LDFLAGS=""
+AC_CHECK_LIB(ncurses,setupterm,LIBS="$LIBS -lncurses -ltinfo")
 AC_CHECK_FUNC(tgetent, , [AC_MSG_ERROR([Could not find tgetent; are you missing a curses/ncurses library?])])
+AC_CHECK_FUNCS(setupterm)
+CFLAGS="$old_CFLAGS"
+LDFLAGS="$old_LDFLAGS"
 KRB5_AC_INET6
-AC_CHECK_FUNCS(setupterm)
 AC_CHECK_HEADER(termios.h,AC_DEFINE(USE_TERMIO,1,[Define if termio should be used]) ac_termio=1)
 if test -z "$ac_termio"; then
 AC_CHECK_HEADER(termio.h,AC_DEFINE(SYSV_TERMIO,1,[Define if SysV termio interface is found]),ac_sysv_termio=1)
@@ -101,11 +101,7 @@
 dnl from old telnetd/configure.in
 dnl
 dnl AC_PROG_INSTALL
-AC_CHECK_LIB(termcap,main,AC_DEFINE(TERMCAP)
-LIBS="$LIBS -ltermcap",
-  AC_CHECK_LIB(curses,setupterm,LIBS="$LIBS -lcurses",
-  AC_CHECK_LIB(ncurses,setupterm,LIBS="$LIBS -lncurses")
-))
+AC_CHECK_LIB(ncurses,setupterm,LIBS="$LIBS -lncurses -ltinfo")
 dnl ... whole termios.h/termio.h/NO_CC_T thing again ...
 AC_HEADER_TIME
 dnl KRB5_AC_INET6
