diff -Nur krb5-1.0.5/src/appl/telnet/telnetd/utility.c krb5-1.0.5.pld/src/appl/telnet/telnetd/utility.c
--- krb5-1.0.5/src/appl/telnet/telnetd/utility.c	Sat Feb  7 04:41:48 1998
+++ krb5-1.0.5.pld/src/appl/telnet/telnetd/utility.c	Fri Feb 19 13:54:43 1999
@@ -90,7 +90,7 @@
 stilloob(s)
     int	s;		/* socket number */
 {
-    static struct timeval timeout = { 0 };
+    static struct timeval timeout = { 0, 0 };
     fd_set	excepts;
     int value;
 
