--- krb5-1.6/src/tests/resolve/Makefile.in~	2006-10-14 01:54:24.000000000 +0200
+++ krb5-1.6/src/tests/resolve/Makefile.in	2007-03-31 13:19:53.138858011 +0200
@@ -22,9 +22,9 @@
 	$(CC_LINK) -o $@ fake-addrinfo-test.o $(SUPPORT_LIB) $(LIBS)
 
 check:: resolve addrinfo-test fake-addrinfo-test
-	$(RUN_SETUP) $(VALGRIND) ./resolve
-	$(RUN_SETUP) $(VALGRIND) ./addrinfo-test -p telnet
-	$(RUN_SETUP) $(VALGRIND) ./fake-addrinfo-test -p telnet
+	if [ "$(OFFLINE)" = no ]; then $(RUN_SETUP) $(VALGRIND) ./resolve ep09.pld-linux.org ; fi
+	if [ "$(OFFLINE)" = no ]; then $(RUN_SETUP) $(VALGRIND) ./addrinfo-test -p telnet ; fi
+	if [ "$(OFFLINE)" = no ]; then $(RUN_SETUP) $(VALGRIND) ./fake-addrinfo-test -p telnet ; fi
 
 install::
 
--- krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test~	2007-11-21 17:52:04.000000000 +0100
+++ krb5-1.6.3/src/plugins/kdb/db2/libdb2/test/run.test	2007-11-21 17:58:16.000000000 +0100
@@ -12,17 +12,7 @@
 	TMP2=${TMPDIR-.}/t2
 	TMP3=${TMPDIR-.}/t3
 
-	if [ \! -z "$WORDLIST" -a -f "$WORDLIST" ]; then
-		DICT=$WORDLIST
-	elif [ -f /usr/local/lib/dict/words ]; then
-		DICT=/usr/local/lib/dict/words
-	elif [ -f /usr/share/dict/words ]; then
-		DICT=/usr/share/dict/words
-	elif [ -f /usr/dict/words ]; then
-		DICT=/usr/dict/words
-	elif [ -f /usr/share/lib/dict/words ]; then
-		DICT=/usr/share/lib/dict/words
-	elif [ -f $srcdir/../test/dictionary ]; then
+	if [ -f $srcdir/../test/dictionary ]; then
 		DICT=`cd $srcdir/../test && pwd`/dictionary
 	else
 		echo 'run.test: no dictionary'
--- krb5-1.9.1/src/lib/krb5/krb/Makefile.in~	2010-12-03 01:05:44.000000000 +0100
+++ krb5-1.9.1/src/lib/krb5/krb/Makefile.in	2011-08-03 00:38:37.538243651 +0200
@@ -431,7 +431,9 @@
 	$(RUN_SETUP) $(VALGRIND) ./t_etypes
 
 check-pytests:: t_expire_warn
-	$(RUNPYTEST) $(srcdir)/t_expire_warn.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_expire_warn.py $(PYTESTFLAGS) ; \
+	fi
 
 clean::
 	$(RM) $(OUTPRE)t_walk_rtree$(EXEEXT) $(OUTPRE)t_walk_rtree.$(OBJEXT) \
--- krb5-1.9.1/src/kdc/Makefile.in~	2010-11-01 20:49:29.000000000 +0100
+++ krb5-1.9.1/src/kdc/Makefile.in	2011-08-03 00:50:14.395525895 +0200
@@ -66,7 +66,9 @@
 	$(RM) test.out
 
 check-pytests::
-	$(RUNPYTEST) $(srcdir)/t_workers.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_workers.py $(PYTESTFLAGS) ; \
+	fi
 
 install::
 	$(INSTALL_PROGRAM) krb5kdc ${DESTDIR}$(SERVER_BINDIR)/krb5kdc
--- krb5-1.9.1/src/appl/gss-sample/Makefile.in~	2010-12-03 01:05:44.000000000 +0100
+++ krb5-1.9.1/src/appl/gss-sample/Makefile.in	2011-08-03 01:03:37.904570242 +0200
@@ -45,7 +45,9 @@
 	$(RM) gss-server gss-client
 
 check-pytests::
-	$(RUNPYTEST) $(srcdir)/t_gss_sample.py $(PYTESTFLAGS)
+	if [ "$(OFFLINE)" = no ]; then \
+	$(RUNPYTEST) $(srcdir)/t_gss_sample.py $(PYTESTFLAGS) ; \
+	fi
 
 install-unix::
 	$(INSTALL_PROGRAM) gss-client $(DESTDIR)$(CLIENT_BINDIR)/gss-client
