diff -urN krb5-1.6/doc/Makefile krb5-1.6.doc/doc/Makefile
--- krb5-1.6/doc/Makefile	2006-12-21 23:07:10.000000000 +0100
+++ krb5-1.6.doc/doc/Makefile	2007-03-30 21:53:16.554101078 +0200
@@ -36,7 +36,7 @@
 admin-guide-full:: admin-guide admin-guide-info admin-guide-html
 
 .PHONY: admin-guide
-admin-guide:: admin-guide.ps
+admin-guide:: admin-guide.pdf
 
 admin-guide.ps: $(ADMIN_DEPS)
 	$(DVI) admin.texinfo
@@ -61,7 +61,7 @@
 install-guide-full:: install-guide install-guide-info install-guide-html
 
 .PHONY: install-guide
-install-guide:: install-guide.ps
+install-guide:: install-guide.pdf
 
 install-guide.ps: $(INSTALL_DEPS)
 	$(DVI) install.texinfo
@@ -86,7 +86,7 @@
 user-guide-full:: user-guide user-guide-info user-guide-html
 
 .PHONY: user-guide
-user-guide:: user-guide.ps
+user-guide:: user-guide.pdf
 
 user-guide.ps: $(USER_GUIDE_DEPS)
 	$(DVI) user-guide.texinfo
@@ -122,12 +122,15 @@
 krb425-guide-full:: krb425-guide krb425-guide-info krb425-guide-html
 
 .PHONY: krb425-guide
-krb425-guide:: krb425-guide.ps
+krb425-guide:: krb425-guide.pdf
 
 krb425-guide.ps: $(KRB425_DEPS)
 	$(DVI) krb425.texinfo
 	$(DVIPS) krb425
 
+krb425-guide.pdf: krb425-guide.ps
+	$(PSPDF) krb425-guide.ps krb425-guide.pdf
+
 .PHONY: krb425-guide-html
 krb425-guide-html:: krb425.html
 
diff -urN krb5-1.6/doc/api/Makefile krb5-1.6.doc/doc/api/Makefile
--- krb5-1.6/doc/api/Makefile	2005-08-29 21:22:52.000000000 +0200
+++ krb5-1.6.doc/doc/api/Makefile	2007-03-30 21:16:19.067733731 +0200
@@ -1,4 +1,4 @@
-.SUFFIXES: .tex .dvi .ps
+.SUFFIXES: .tex .dvi .ps .pdf
 
 SHELL=/bin/sh
 
@@ -8,7 +8,7 @@
 
 DESTEX= libdes.tex
 
-all: library.ps libdes.ps
+all: library.pdf libdes.pdf
 
 
 libdes.dvi: $(DESTEX) $(STYLES)
@@ -19,7 +19,7 @@
 	rm -f *.toc *.log *.idx *.ind *.aux *.ilg lib1.stamp
 
 really-clean: clean
-	rm -f *.dvi *.ps
+	rm -f *.dvi *.ps *.pdf
 
 
 library.dvi:  lib1.stamp $(LIBTEX) $(STYLES)
@@ -31,10 +31,22 @@
 	makeindex -s krb5.ist library.idx
 	date > lib1.stamp
 
+libdes.pdf: $(DESTEX) $(STYLES)
+	touch libdes.ind
+	pdflatex libdes.tex
+	pdflatex libdes.tex
+	pdflatex libdes.tex
+
+library.pdf: $(LIBTEX) $(STYLES)
+	touch library.ind
+	pdflatex library.tex
+	makeindex -s krb5.ist library.idx
+	pdflatex library.tex
+	pdflatex library.tex
+
 .tex.dvi:
 	latex $*
 
 
 .dvi.ps:
 	dvips $*.dvi -o
-
diff -urN krb5-1.6/doc/api/library.tex krb5-1.6.doc/doc/api/library.tex
--- krb5-1.6/doc/api/library.tex	2005-08-29 21:22:52.000000000 +0200
+++ krb5-1.6.doc/doc/api/library.tex	2007-03-30 20:32:44.646235755 +0200
@@ -1,7 +1,7 @@
-\documentclass[twoside]{article}
+\documentclass{article}
 \usepackage{fixunder,functions,fancyheadings}
 \usepackage{krb5idx}
-%\usepackage{hyperref}
+\usepackage{hyperref}
 
 %\hypersetup{letterpaper,
 %    bookmarks=true,
diff -urN krb5-1.6/doc/implement/Makefile krb5-1.6.doc/doc/implement/Makefile
--- krb5-1.6/doc/implement/Makefile	2005-08-29 21:22:52.000000000 +0200
+++ krb5-1.6.doc/doc/implement/Makefile	2007-03-30 21:15:21.716465469 +0200
@@ -5,7 +5,7 @@
 	kdb-i.tex encrypt-i.tex cksum-i.tex crc-32-i.tex implement.ind
 
 
-all: implement.ps
+all: implement.pdf
 
 
 implement.ps: implement.dvi
@@ -20,6 +20,13 @@
 	latex implement.tex
 	rm implement.ind
 
+implement.pdf: $(LIBTEX) $(STYLES)
+	touch implement.ind
+	pdflatex implement.tex
+	makeindex implement.idx
+	pdflatex implement.tex
+	pdflatex implement.tex
+
 clean:
 	rm -f *.toc *.log *.idx *.ind *.aux *.ilg
 
diff -urN krb5-1.6/doc/implement/implement.tex krb5-1.6.doc/doc/implement/implement.tex
--- krb5-1.6/doc/implement/implement.tex	2005-08-29 21:22:52.000000000 +0200
+++ krb5-1.6.doc/doc/implement/implement.tex	2007-03-30 20:32:44.646235755 +0200
@@ -1,4 +1,8 @@
-\documentstyle[fixunder,functions,twoside,fancyheadings]{article}
+\documentclass{article}
+\usepackage{fixunder}
+\usepackage{functions}
+\usepackage{fancyheadings}
+\usepackage{hyperref}
 \setlength{\oddsidemargin}{0in}
 \setlength{\evensidemargin}{2.00in}
 \setlength{\marginparsep}{0.05in}
diff -urN krb5-1.6/doc/kadm5/Makefile krb5-1.6.doc/doc/kadm5/Makefile
--- krb5-1.6/doc/kadm5/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ krb5-1.6.doc/doc/kadm5/Makefile	2007-03-30 21:38:08.254340024 +0200
@@ -0,0 +1,31 @@
+.SUFFIXES: .tex .dvi .ps
+
+STYLES= fullpage.sty rcsid.sty
+
+all: adb-unit-test.pdf api-funcspec.pdf api-server-design.pdf api-unit-test.pdf
+
+adb-unit-test.pdf: adb-unit-test.tex $(STYLES)
+	pdflatex adb-unit-test.tex
+	pdflatex adb-unit-test.tex
+	pdflatex adb-unit-test.tex
+
+api-funcspec.pdf: api-funcspec.tex $(STYLES)
+	pdflatex api-funcspec.tex
+	pdflatex api-funcspec.tex
+	pdflatex api-funcspec.tex
+
+api-server-design.pdf: api-server-design.tex $(STYLES)
+	pdflatex api-server-design.tex
+	pdflatex api-server-design.tex
+	pdflatex api-server-design.tex
+
+api-unit-test.pdf: api-unit-test.tex $(STYLES)
+	pdflatex api-unit-test.tex
+	pdflatex api-unit-test.tex
+	pdflatex api-unit-test.tex
+
+clean:
+	rm -f *.toc *.log *.idx *.ind *.aux *.ilg
+
+really-clean: clean
+	rm -f *.dvi *.ps
diff -urN krb5-1.6/doc/man2html krb5-1.6.doc/doc/man2html
--- krb5-1.6/doc/man2html	2005-08-26 01:15:24.000000000 +0200
+++ krb5-1.6.doc/doc/man2html	2007-03-30 21:18:01.713583185 +0200
@@ -1,4 +1,4 @@
-#!/usr/athena/bin/perl
+#!/usr/bin/perl
 #!/usr/local/bin/perl
 ##---------------------------------------------------------------------------##
 ##  File:
