correct a typo in the krb5.conf man page

Index: src/config-files/krb5.conf.M
===================================================================
--- src/config-files/krb5.conf.M	(revision 19507)
+++ src/config-files/krb5.conf.M	(working copy)
@@ -600,7 +600,7 @@
 objects used for starting the Kerberos servers. This value is used if no
 service password file is mentioned in the configuration section under dbmodules.
 
-.IP ldap_server
+.IP ldap_servers
 This LDAP specific tag indicates the list of LDAP servers. The list of LDAP servers
 is whitespace-separated. The LDAP server is specified by a LDAP URI.
 This value is used if no LDAP servers are mentioned in the configuration
@@ -641,7 +641,7 @@
 This LDAP specific tag indicates the file containing the stashed passwords for the
 objects used for starting the Kerberos servers.
 
-.IP ldap_server
+.IP ldap_servers
 This LDAP specific tag indicates the list of LDAP servers. The list of LDAP servers
 is whitespace-separated. The LDAP server is specified by a LDAP URI.
 
