--- krb5-1.15/src/config/shlib.conf.orig	2016-12-01 23:31:24.000000000 +0100
+++ krb5-1.15/src/config/shlib.conf	2017-02-19 10:36:34.400096062 +0100
@@ -544,8 +544,8 @@
 
 if test "${MAKE_SHLIB_COMMAND}" = "x" ; then
   if test "${INIT_FINI_PREP}" != ":"; then
-    MAKE_SHLIB_COMMAND="${INIT_FINI_PREP} && ${LDCOMBINE} -o \$@ \$\$objlist \$(SHLIB_EXPFLAGS) \$(LDFLAGS) ${LDCOMBINE_TAIL}"
+    MAKE_SHLIB_COMMAND="${INIT_FINI_PREP} && ${LDCOMBINE} -o \$@ \$\$objlist \$(LDFLAGS) \$(SHLIB_EXPFLAGS) ${LDCOMBINE_TAIL}"
   else
-    MAKE_SHLIB_COMMAND="${LDCOMBINE} -o \$@ \$\$objlist \$(SHLIB_EXPFLAGS) \$(LDFLAGS) ${LDCOMBINE_TAIL}"
+    MAKE_SHLIB_COMMAND="${LDCOMBINE} -o \$@ \$\$objlist \$(LDFLAGS) \$(SHLIB_EXPFLAGS) ${LDCOMBINE_TAIL}"
   fi
 fi
