diff -Nur krb5-1.0.5/src/lib/rpc/svc.c krb5-1.0.5.pld/src/lib/rpc/svc.c
--- krb5-1.0.5/src/lib/rpc/svc.c	Sat Feb  7 04:45:12 1998
+++ krb5-1.0.5.pld/src/lib/rpc/svc.c	Wed Feb 17 16:18:02 1999
@@ -40,7 +40,11 @@
  *
  * Copyright (C) 1984, Sun Microsystems, Inc.
  */
-
+#ifdef __linux__
+# ifndef _GNU_SOURCE
+# define _GNU_SOURCE
+# endif
+#endif
 #include <sys/errno.h>
 #include <rpc/rpc.h>
 #include <rpc/pmap_clnt.h>
