Use an in-memory ccache to silence a compiler warning.
--- krb5-1.12.1/src/slave/kprop.c.orig	2014-01-16 01:44:15.000000000 +0100
+++ krb5-1.12.1/src/slave/kprop.c	2014-03-13 17:07:46.637105824 +0100
@@ -187,9 +187,9 @@
 void get_tickets(context)
     krb5_context context;
 {
-    char   buf[BUFSIZ], *def_realm;
+    char   *def_realm;
     krb5_error_code retval;
-    static char tkstring[] = "/tmp/kproptktXXXXXX";
+    char tkstring[] = "MEMORY:_kproptkt";
     krb5_keytab keytab = NULL;
 
     /*
@@ -230,20 +230,17 @@
 #endif
 
     /*
-     * Initialize cache file which we're going to be using
+     * Initialize an in-memory cache for temporary use
      */
-    (void) mktemp(tkstring);
-    snprintf(buf, sizeof(buf), "FILE:%s", tkstring);
-
-    retval = krb5_cc_resolve(context, buf, &ccache);
+    retval = krb5_cc_resolve(context, tkstring, &ccache);
     if (retval) {
-        com_err(progname, retval, _("while opening credential cache %s"), buf);
+        com_err(progname, retval, _("while opening credential cache %s"), tkstring);
         exit(1);
     }
 
     retval = krb5_cc_initialize(context, ccache, my_principal);
     if (retval) {
-        com_err(progname, retval, _("when initializing cache %s"), buf);
+        com_err(progname, retval, _("when initializing cache %s"), tkstring);
         exit(1);
     }
 
