--- krb5-1.18.5/src/build-tools/krb5-config.in.orig	2024-02-09 18:34:31.674890065 +0100
+++ krb5-1.18.5/src/build-tools/krb5-config.in	2024-02-09 18:49:52.946565774 +0100
@@ -33,7 +33,8 @@ includedir=@includedir@
 libdir=@libdir@
 CC_LINK='@CC_LINK@'
 KDB5_DB_LIB=@KDB5_DB_LIB@
-LDFLAGS='@LDFLAGS@'
+LDFLAGS=
+CFLAGS=
 RPATH_FLAG='@RPATH_FLAG@'
 PROG_RPATH_FLAGS='@PROG_RPATH_FLAGS@'
 PTHREAD_CFLAGS='@PTHREAD_CFLAGS@'
